CMakeLists.txt

```diff
@@ -65,7 +65,8 @@ endif()
 
 message(STATUS "OTBR package name: ${OTBR_NAME}")
 
-if(NOT OTBR_VERSION)
+set(OTBR_VERSION "" CACHE STRING "OTBR Package Version")
```

Comment:
Basically doing the same as OT_PACKAGE_VERSION in third_party/openthread/repo/CMakeLists.txt